composer.json

```diff
@@ -32,7 +32,8 @@
     "znk3r/html_quickform": "4.0.2",
     "friendsofpear/pear_exception": "0.0.*",
     "znk3r/html_common": "*",
-    "pear/pear": "^1.10"
+    "pear/pear": "^1.10",
+    "excelwebzone/recaptcha-bundle": "1.5.13"
```

Comment:
Why not using a caret?

Reply:
j'ai mis cela dans le commentaire du commit, https://github.com/afup/web/pull/948/commits/2d9e5343bf1265adff810ad6d55e67270e27973e
on a des soucis avec notre version de twig qu'il faudrait mettre à jour

Reply:
Du coup peut-être à modifier quand on passera #939